src/components/auth/node.tsx

```diff
@@ -65,13 +65,40 @@ export function Node({
         )
 
       case 'checkbox':
-      case 'button':
-        triggerSentry({
-          message: `kratos: tried to render input node which is not supported atm: ${attributes.type}`,
-        })
-        return null
+      case 'button': {
+        const label = node.meta.label?.id
+          ? getKratosMessageString(
+              node.meta.label.id,
+              strings.auth.messages,
+              'Undefined'
+            )
+          : 'Could not find translation'
```

Comment:
`'Undefined'` as a fallback makes no sense here. Maybe use `node.meta.label.text`?
Also the label should not contain an error message. I guess if `node.meta.label` is actually not defined we should not render a button at all (any optinally trigger a sentry error).